src/test/scala/performanceanalysis/LogParserActorSpec.scala

```diff
@@ -8,6 +8,11 @@ import performanceanalysis.logreceiver.alert.AlertRuleActorCreator
 import performanceanalysis.server.Protocol.Rules.{AlertRule, Threshold, Action => RuleAction}
 import performanceanalysis.server.Protocol._
 
+import scala.concurrent.duration.Duration
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/scala-academy/performance-analysis/file/2632938597/issues/source?bid=3220915&fileBranchId=3304665#l11)